15314f1ddb882700dc9fabc5ca961943/dictionary/x_illu2_illumio_illumio_label_dimension.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?><database>
+    <element db_object_id="da7f8d4897200a10b7daf896f053af39" label="Illumio Label Dimension" max_length="40" name="x_illu2_illumio_illumio_label_dimension" type="collection">
+        <element label="Display Name (plural)" mandatory="true" max_length="100" name="display_name_plural" type="string"/>
+        <element display="true" label="Display Name (singular)" mandatory="true" max_length="100" name="display_name_singular" type="string"/>
+        <element label="Href" max_length="100" name="href" read_only="true" type="string"/>
+        <element label="Illumio PCE Configuration" mandatory="true" max_length="32" name="illumio_pce_configuration" reference="x_illu2_illumio_illumio_pce_conf" type="reference"/>
+        <element label="isDeleted" max_length="40" name="isdeleted" read_only="true" type="boolean"/>
```

Comment:
nit: this is a boolean, setting max length to 40 is not needed.

Reply:
The max length attribute is added to each column of a ServiceNow table by default. It is a mandatory attribute and the default length selected by ServiceNow is 40.

Reply:
Yes, so setting max length to 40 is not required for a boolean field. Please change accordingly.

Reply:
Unfortunately, that doesn’t fall in our control as per the platform’s behaviour ServiceNow adds max_length to each created columns by itself.